docs/content/cheatsheet.md

```diff
@@ -0,0 +1,1159 @@
+    <target>
+    ```
+  - ```md
+    <https://jupyterbook.org>
+    ```
+  - <https://jupyterbook.org>
+* - ```md
+    [text][key]
+    ```
+  - ```md
+    [Jupyter Book][intro_page]
+
+    [intro_page]: https://jupyterbook.org
+    ```
+  - [Jupyter Book][intro_page]
+
+    [intro_page]: https://jupyterbook.org
+``````
+
+## Lists
+
+### Ordered List
+
+``````{list-table}
+:header-rows: 1
+:widths: 20 20
+
+* - Example
+  - Result
+* - ```md
+    1. First item
+    2. Second item
+      1. First sub-item
+    ```
+  - 1. First item
+    2. Second item
+      1. First sub-item
+* - ```md
+    1. First item
+    2. Second item
+      * First sub-item
+    ```
+  - 1. First item
+    2. Second item
+      * First subitem
+``````
+
+### Unordered List
+
+``````{list-table}
+:header-rows: 1
+:widths: 20 20
+
+* - Example
+  - Result
+* - ```md
+    * First item
+    * Second item
+      * First subitem
+    ```
+  - * First item
+    * Second item
+      * First subitem
+* - ```md
+    * First item
+      1. First subitem
+      2. Second subitem
+    ```
+  - * First item
+      1. First subitem
+      2. Second subitem
+``````
+
+
+## Tables
+
+``````{list-table}
+:header-rows: 1
+:widths: 20 20 20
+
+* - Text
+  - Example
+  - Result
+* - ```md
+    | a    | b    |
+    | :--- | ---: |
+    | c    | d    |
+    ```
+  - ```md
+    |    Training   |   Validation   |
+    | :------------ | -------------: |
+    |        0      |        5       |
+    |     13720     |      2744      |
+    ```
+  - |    Training   |   Validation   |
+    | :------------ | -------------: |
+    |        0      |        5       |
+    |     13720     |      2744      |
+* - ````md
+     ```{list-table}
+    :header-rows: 1
+
+    * - Col1
+      - Col2
+    * - Row1 under Col1
+      - Row1 under Col2
+    * - Row2 under Col1
+      - Row2 under Col2
+    ```
+    ````
+  - ````md
+     ```{list-table}
+    :header-rows: 1
+    :name: example-table
+
+    * - Training
+      - Validation
+    * - 0
+      - 5
+    * - 13720
+      - 2744
+    ```
+    ````
+  - ```{list-table}
+    :header-rows: 1
+    :name: example-table
+
+    * - Training
+      - Validation
+    * - 0
+      - 5
+    * - 13720
+      - 2744
+    ```
+* - ````md
+     ```{list-table} title
+    :header-rows: 1
+
+    * - Col1
+      - Col2
+    * - Row1 under Col1
+      - Row1 under Col2
+    * - Row2 under Col1
+      - Row2 under Col2
+    ```
+    ````
+  - ````md
+     ```{list-table} Table with a title
+    :header-rows: 1
+
+    * - Training
+      - Validation
+    * - 0
+      - 5
+    * - 13720
+      - 2744
+    ```
+    ````
+  - ```{list-table} Table with a title
+    :header-rows: 1
+
+    * - Training
+      - Validation
+    * - 0
+      - 5
+    * - 13720
+      - 2744
+    ```
+``````
+
+### Referencing Tables
+
+```{note}
+To add a label to your table simply include a `:name:` parameter followed by the label of your table. See example above.
+```
+
+``````{list-table}
```

Comment:
in the rendered page it seems like there are two list-tables that are duplicates of each other

Reply:
Are you referring to the last two items in the table? I can remove the last item and add the title under the second table example.

Reply:
Ah whoops, I see now that one of them has a title while the other doesn't, so they are correctly two different tables, not copies of the same one.